### PR TITLE
Dynamic pricing page

### PR DESCRIPTION
**Pull request by henrikingo**
Blocker:Need to verify c7g produces stable results and we don't need dedicated instances.

### CODE REVIEW

**Comment by henrikingo** [comment]
@mfleming You may want to run this for yourself to see the page.

Short term we probably want to remove the CPUH/DSI stuff and add it back when we're actually ready to sell it.

File: frontend/src/components/PricingPage.jsx [hunk 1]
```diff
@@ -1,29 +1,185 @@
+import { useState } from 'react';
+
 export const PricingPage = () => {
+  const businessPriceFloor = 10;
+  const businessPricePerHead = 20;
```
Comment (mfleming) [inline frontend/src/components/PricingPage.jsx:5]: I actually think it's kinda cool the price info is in the code :)

**Review by mfleming — APPROVED** [review]
It's a minor point but there's some rounding missing somewhere:
image

File: frontend/src/components/PricingPage.jsx (hunk 1 above)
Comment (henrikingo) [reply frontend/src/components/PricingPage.jsx:5]: Agree!

**Pull request merged**